### PR TITLE
Telemetry and APRS Callsign + Passcode Update

### PR DESCRIPTION
**Pull request by kb1lqc**
- Cleaned up telemetry.py upper() statements
- Forced upper() on telemetry callsign URL paramerter get() calls
- Changed callsign default value from "None" to "%" which passes an upper() call and is a valid SQL wildcard
- Removed SQL string generation for callsign = None
- No longer uses login.ini configuration file for APRS, APRS-IS callsign located in aprs.ini now
- Implemented a basic APRS-IS passcode generator. This automatically generates a passcode to protect against simple poor passcode entry and helps remove an extra configuration file.
- Checks for callsign= "REPLACEME" and warns user while not attempting a connection which will not work as intended.
- Changed telemetry rate to 60 seconds
- Changed socket.error exception checks over to ERRORIO in an attempt to gracefully fail better.
- Update APRS/readme.md to incorporate new configuration necessary to run with passcode generator.

This is related to #91 and should fix this issue. The code should be tested to work for all who properly configure the APRS application.

### CODE REVIEW

**Comment by kb1lqc** [comment]
Please move this into smaller chunks. It's really really really hard to follow and understand with lots of little errors. If we want this to go out fast then make small bit-size pull requests that we can focus on and optimize to release one at a time.

File: Applications/APRS/aprs.ini [hunk 1]
```diff
@@ -3,9 +3,10 @@ HOST=127.0.0.1
 PORT=8001
 
 [APRSIS]
+CALLSIGN=REPLACEME
```
Comment (kb1lqd) [inline Applications/APRS/aprs.ini:6]: Why is this callsign here? Is this for passcode to APRS-IS?

Comment (kb1lqd) [reply Applications/APRS/aprs.ini:6]: Oh is it this "No longer uses login.ini configuration file for APRS, APRS-IS callsign located in aprs.ini now"

File: Applications/APRS/aprs.py [hunk 2]
```diff
@@ -260,7 +256,7 @@ def sendPositions(stations, socket):
                 try:
                     socket.sendall(positionString)
 
-                except socket.error as e:
+                except IOError as e:
```
Comment (kb1lqd) [inline Applications/APRS/aprs.py:259]: Why will this fail more gracefully than socket.error? What is IOERROR?

File: Applications/APRS/aprs.ini (hunk 1 above)
Comment (kb1lqd) [reply Applications/APRS/aprs.ini:6]: Disregard.

File: proxy/proxy.py [hunk 3]
```diff
@@ -237,8 +237,8 @@ def proxy():
                 # Required
                 raise StandardError("Missing 'callsign' parameter")
             else:
-                # Ensure callsign value is a string and all uppercase
-                callsign = str(callsign).upper()
+                # Ensure callsign value is a stringe
```
Comment (kb1lqd) [inline proxy/proxy.py]: Non Blocking: Typo, should be string not stringe?

**Comment by kb1lqd** [comment]
* Confirmed proxy forces callsign to UPPERCASE
* Confirmed telemetry forces callsign to UPPERCASE
* Confirmed APRS detects and soft errors on README left in INI file
* Confirmed APRS forces UPPERCASE
* Confirmed APRS works and connects, verified position reports updating on APRS.fi with a GPS enabled Faraday unit

File: Applications/APRS/readme.md [hunk 4]
```diff
@@ -34,4 +32,4 @@ With these running simply run `aprs.py` and the application will startup. It con
  
 ## Correctly Running APRS Application
 ![APRS application running with fixed GPS location](images/APRS_Running_FixedPosition.exe.png "APRS Application")
-The application running above is nominal and connected to a `KB1LQC-1` Faraday radio through [Proxy](../../Proxy) which doesn't have a GPS. Since GPS Fix is therefore zero APRS simply warns that the unit has a bad GPS Fix. However, since we know it's configured with a fixed location in Flash memory this can be disregarded.
\ No newline at end of file
+The application running above is nominal and connected to a `KB1LQC-1` Faraday radio through [Proxy](../../Proxy) which doesn't have a GPS. Since GPS Fix is therefore zero APRS simply warns that the unit has a bad GPS Fix. However, since we know it's configured with a fixed location in Flash memory this can be disregarded.
```
Comment (kb1lqd) [inline Applications/APRS/readme.md]: Blocking: Image not aligned as shown below. Text should be below the image not in-line.

image

**Review by kb1lqd — CHANGES_REQUESTED** [review]
Most are questions/non-blocking. A few blocking.

File: Applications/APRS/readme.md (hunk 4 above)
Comment (kb1lqc) [reply Applications/APRS/readme.md]: Fixed

File: Applications/APRS/aprs.py (hunk 2 above)
Comment (kb1lqc) [reply Applications/APRS/aprs.py:259]: `socket.error` is a [child of IOError](https://docs.python.org/2/library/exceptions.html#exceptions.IOError). This means IOError is a little more broad and sweeping. Additionally for some reason `socket.error` was not being found [when used in aprs.py](https://github.com/FaradayRF/Faraday-Software/issues/91#issuecomment-281220467). Using IOError fixed this as it's a built-in exception. I can investigate more as to why the socket module wasn't letting me do this but I figured this was an acceptable fix.

File: proxy/proxy.py (hunk 3 above)
Comment (kb1lqc) [reply proxy/proxy.py]: Fixed

**Review by kb1lqd — APPROVED** [review]
